packages/block/src/block.ts

```diff
@@ -137,6 +147,65 @@ export class Block {
     return new Block(header, transactions, uncleHeaders, opts)
   }
 
+  /**
+   * Creates a new block object from Ethereum JSON RPC.
+   *
+   * @param blockParams - Ethereum JSON RPC of block (eth_getBlockByNumber)
+   * @param uncles - Optional list of Ethereum JSON RPC of uncles (eth_getUncleByBlockHashAndIndex)
+   * @param options - An object describing the blockchain
+   */
+  public static fromRPC(blockData: any, uncles?: any[], opts?: BlockOptions) {
+    return blockFromRpc(blockData, uncles, opts)
+  }
+
+  /**
+   *  method to retrieve a block from the provider to use in the VM
```

Comment:
method -> Method

Description shouldn't be targeted towards VM usage, this is a standalone library.

One additional newline below

Reply:
Yep, forgot to update when I copied it over